jest.config.js

```diff
@@ -0,0 +1,10 @@
+const commonCofig = require('@folio/stripes-acq-components/jest.config');
+
+module.exports = {
+  ...commonCofig,
+  testMatch: ['**/FindPOLine/**/?(*.)test.{js,jsx}'],
+  collectCoverageFrom: [
+    '**/FindPOLine/**/*.{js,jsx}',
+    '!**/node_modules/**',
+  ],
```

Comment:
IMHO, `'!**/test/**',` is missed. We don't have take into account coverage on bigtest test files

Reply:
https://github.com/folio-org/ui-plugin-find-po-line/pull/79